### PR TITLE
SAK-42090 Associated labels for fields properly

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-42090

### CODE REVIEW

File: rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-grading-comment.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class SakaiRubricGradingComment extends RubricsElement {
         <div class="arrow"></div>
         <div class="popover-title" style="display: flex;">
           <div style="flex: auto;">
-            <label class="criterion-title" for="">
+            <label class="criterion-title">
```
Comment (ottenhoff) [inline rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-grading-comment.js]: so is this not a label for anything? should just be a span or div?

Comment (adrianfish) [reply rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-grading-comment.js]: I thought that, but I just thought it best to leave it alone as it wasn't part of this ticket. I can dig in a bit more.

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criterion-edit.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakaiproject/sakai/pullRequest?prid=4487057&bid=15137290)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criterion-edit.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakaiproject/sakai/pullRequest?prid=4487057&bid=15137290)

**Pull request merged**